app/Core/Ldap/User.php

```diff
@@ -116,12 +127,18 @@ protected function getGroups(Entry $entry, $username)
      */
     protected function getRole(array $groupIds)
     {
-        if (! $this->hasGroupsConfigured()) {
-            return null;
-        }
 
         // Init with smallest role
         $role = Role::APP_USER ;
+        
+        if (! $this->hasGroupsConfigured()) {
+            if ($this->userDefaultRole()) {
```

Comment:
This function is not really useful anymore. You can just use the constant here: `if (LDAP_USER_DEFAULT_ROLE_MANAGER) {...`